### DISCUSSION TITLE
scRNA cross-species integration using Seurat

### DISCUSSION THREAD

**Discussion by priyamaddhesiya** [satijalab/seurat#4245] [Q&A] [answered]
Hi, I am trying to integrate zebrafish and mouse genome using Seurat. I am getting an error while running the anchoring step (FindIntegrationAnchors). I have the orthologous table between the two species obtained from biomart however I don't understand at what step should I upload it to reannotate the genes in order to get common features? 
Many thanks, hope to get your suggestions!

**Comment by timoast** [comment, marked as answer]
To perform cross-species integration you need to ensure that the genes have the same name across the different Seurat objects to be integrated. If you have a list of gene orthologs, you should rename the genes in each count matrix with the ortholog name before creating the Seurat object. We don't currently support renaming features in an existing assay. If you want to retain the original species name, you could add the original count matrix as a separate assay.

**Reply by lvclark** [reply]
It would be really helpful if you did support renaming genes in a Seurat object!

**Comment by ccy-663** [comment]
I've met exactly the same problem with you. I wonder if you have found a solution.
